nixos/modules/services/games/minecraft-server.nix

```diff
@@ -20,6 +82,12 @@ in
         '';
       };
 
+      eula = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Must agree to EULA to run minecraft server";
+      };
```

Comment:
I think an assertion with the message that the user has to set this option to `true` to have it work is appropriate. The same message you get when you start the server without this set to true.